phpstan.neon

```
@@ -28,6 +28,8 @@ parameters:
 		- phpstan-bootstrap.php
 	ignoreErrors:
 		- '/Variable \$\w+ might not be defined\./'
+		# resolve later with translator interface generics
+		- '#Method Mautic\\PageBundle\\Model\\PageModel::getTranslatedEntity\(\) return type has no value type specified in iterable type array#'
```

I put the ignored error here on purpose, so we are aware of the need for generics with translator. Hope it's ok